packages/cart/src/scripts/migration-down.ts

```diff
@@ -0,0 +1,50 @@
+import { Modules } from "@medusajs/modules-sdk";
```

Comment:
comment: These scripts seem to contain a lot of duplicated code with little to no changes across modules. We should consider a refactor to eliminate the amount of boilerplate code

Reply:
I believe we can create utils function to build those script and export them from the modules